ckan/templates-midnight-blue/group/snippets/group_item.html

```diff
@@ -1,64 +1,59 @@
 {#
-Renders a media item for a group. This should be used in a list.
+Renders a card for a group. This should be used in a list.
 
 group - A group dict.
 
 Example:
 
-    <ul class="media-grid">
+    <ul class="group-list">
       {% for group in groups %}
         {% snippet "group/snippets/group_item.html", group=group %}
       {% endfor %}
     </ul>
 #}
+
 {% set type = group.type or 'group' %}
 {% set url = h.url_for(type ~ '.read', id=group.name) %}
 {% block item %}
-<li class="media-item">
+<li class="groups-cards">
+  {% block link %}
+  {% set view_label = _('Open details page for {name}').format(name=group.display_name) %}
+    <a href="{{ url }}" aria-label="{{ view_label }}" data-bs-title="{{ view_label }}" data-bs-toggle="tooltip">
   {% block item_inner %}
   {% block image %}
-    <img src="{{ group.image_display_url or h.url_for_static('/base/images/placeholder-group.png') }}" alt="{{ group.name }}" class="media-image img-fluid">
+    <div class="module-image">
+      <img src="{{ group.image_display_url or h.url_for_static('/base/images/placeholder-group.svg') }}" class="img-fluid" alt="{{ group.name }}" />
+    </div>
   {% endblock %}
   {% block title %}
-    <h2 class="media-heading">{{ group.display_name }}</h2>
-  {% endblock %}
-  {% block description %}
-    {% if group.description %}
-      <p class="media-description">{{ h.markdown_extract(group.description, extract_length=80) }}</p>
-    {% endif %}
+    <h3>{{ group.display_name }}</h3>
   {% endblock %}
   {% block datasets %}
     {% if group.package_count %}
-      <strong class="count">{{ ungettext('{num} Dataset', '{num} Datasets', group.package_count).format(num=group.package_count) }}</strong><br/>
-    {% elif group.package_count == 0 %}
-      <span class="count">{{ _('0 Datasets') }}</span><br/>
+      <span class="count">{{ ungettext('{num} Dataset', '{num} Datasets', group.package_count).format(num=group.package_count) }}</span>
+    {% else %}
+      <span class="count">{{ _('0 Datasets') }}</span>
     {% endif %}
   {% endblock %}
-  {% block members %}
+  {# {% block members %}
```

Comment:
Not sure if you are planning to un-comment it later but this block is needed in the dashboard page (http://localhost:5000/dashboard/groups)